### PR TITLE
added my animation

### PR DESCRIPTION
**Pull request by mintera10**

### CODE REVIEW

File: Art/mintera10/index.html [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+        </div>
+        <div class="puppy-leg front-left"></div>
+        <div class="puppy-leg front-right"></div>
+        <div class="puppy-leg back-left"></div>
+        <div class="puppy-leg back-right"></div>
+    </div>
+</body>
+</html>
+<body>
+    <img src="images/puppy-animation.gif">
```
Comment (LaurelineP) [inline Art/mintera10/index.html:27]: ## Invalid source & Invalid implementation maybe
1. HTML issue: You cannot have 2 bodies in HTML
2. Wrong Contribution approach: it seems like you want to display your own gif within your own implementation:
the gif is to be provided in the folder ( looks like you did it with puppy.gif in mintera10 folder ) 
whereas the source image here is referring to an inner folder images that does not exist.

Please remove the whole second body.

File: Art/mintera10/puppy.gif
Comment (LaurelineP) [inline Art/mintera10/puppy.gif:1]: # Invalid image source
The file seems to be invalid, please try to provide a correct file ( here there are no contents )

**Review by LaurelineP — CHANGES_REQUESTED** [review]
Aloha   @mintera10  🙌
Thanks for your contribution and time !
Before we could merge your contribution, the following feedbacks need to be addressed first.
Please address the given feedbacks

> [!NOTE]
Checked values - are the one that are correctly done
> - Please address all the unchecked elements
> - If you need guidance , please refer to the

> [!TIP]
> If the name of your folder needs to be changed ensure to adjust the `include.js`

## 📌 Feedbacks:
- Please remove the last body on your HTML
- Please a correct file as gif - it seems you've pushed an empty content gif



## 🔹 Reference Acceptance criteria for a contribution
Correctly:
  - [x] Folder name: named after the github user name in `./Art`  
     - [x] contains a `HTML` file: the contribution must have an HTML file
         - [x] has a <link> tag to the CSS file of the project
         - [x] contains the `HTML` content for the animation
         - [x] does not contain a SVG replacing the whole HTML CSS animation requirements
         - [x] does not include any CSS library:  No CSS library
         - [x] does not contain any JS script:  No JS logic should be part of your submission
     - [x] contains a `CSS` file: the contribution must have a CSS file 
     - [x] the contribution includes the animated file (gif) or picture ( jpeg, jpg, png )
     - [x] the pictural representation must not include the browser frame, just the animation
     - [ ] the contribution includes the correct contribution's details in `include.js`
        - [ ]  the relative path should be correct : referring to the correct named folder
        - [x] `author`: the value should be the github name or github
        - [ ] `artName`: the value should be the animation title
        - [ ] `pageLink`: the value should  be the relative path to your project's `HTML` file
        - [ ] `imageLink`: should refer to the project relative path's image 
        - [ ] `githubLink`: the value should be your github profile link
        - [ ] does not have chevrons
        - [ ] does not have comments for each fields

**Review by LaurelineP — CHANGES_REQUESTED** [review]
Aloha  @mintera10  🙌
Thanks for your contribution and time !
Before we could merge your contribution, the following feedbacks need to be addressed first.
Please address the given feedbacks

> [!NOTE]
Checked values - are the one that are correctly done
> - Please address all the unchecked elements
> - If you need guidance , please refer to the

> [!TIP]
> If the name of your folder needs to be changed ensure to adjust the `include.js`

## 📌 Feedbacks:
- Please remove the second HTML body
- Please provide a correct gif file


## 🔹 Reference Acceptance criteria for a contribution
Correctly:
  - [x] Folder name: named after the github user name in `./Art`  
     - [x] contains a `HTML` file: the contribution must have an HTML file
         - [x] has a <link> tag to the CSS file of the project
         - [x] contains the `HTML` content for the animation
         - [x] does not contain a SVG replacing the whole HTML CSS animation requirements
         - [x] does not include any CSS library:  No CSS library
         - [x] does not contain any JS script:  No JS logic should be part of your submission
     - [x] contains a `CSS` file: the contribution must have a CSS file 
     - [x] the contribution includes the animated file (gif) or picture ( jpeg, jpg, png )
     - [ ] the pictural representation must not include the browser frame, just the animation
     - [x] the contribution includes the correct contribution's details in `include.js`
        - [x]  the relative path should be correct : referring to the correct named folder
        - [x] `author`: the value should be the github name or github
        - [x] `artName`: the value should be the animation title
        - [x] `pageLink`: the value should  be the relative path to your project's `HTML` file
        - [ ] `imageLink`: should refer to the project relative path's image 
        - [ ] `githubLink`: the value should be your github profile link
        - [x] does not have chevrons
        - [ ] does not have comments for each fields

**Comment by mintera10** [comment]
oh mb, will change it. Thanks for the feedback ~

**Comment by mintera10** [comment]
can u check it now?

**Comment by mintera10** [comment]
umm i have actually done all the changes, can you check it and merge it if its fine?

**Pull request merged**